client/js/screens/Home/Home.js

```diff
@@ -192,7 +227,26 @@ class Home extends Component {
               title="Log out"
             />
           </View>
-        </NavScrollView>
+        </AnimatedNavScrollView>
+
+        <Animated.View
+          style={{
+            position: 'absolute',
+            opacity: this.state.headerAnimation.x,
+            top: this.state.headerAnimation.y,
+            width: '100%',
+            height: paddingHeight,
+          }}>
+          <VibrancyView
+            blurType="dark"
+            blurAmount={2}
+            style={{
+              width: '100%',
+              height: '100%',
+            }}
```

Comment:
Please move your styles to styles.js. 
`opacity: this.state.headerAnimation.x,
  top: this.state.headerAnimation.y,` are fine where they are.

Reply:
Done